share/commands/agent-policy-server

```diff
@@ -1,10 +1,14 @@
 #!/bin/sh
 # @description displays or set the policy server
 # @man If called without arguments, displays current policy server.
-# @man Sets the epolicy server to the hostname or IP given.
+# @man Sets the policy server to the hostname or IP given.
 # @man +
 # @man *Arguments*:
 # @man +
+# @man + *-t*: trust provided sha256 server ket hash
+# @man +
+# @man + *-p*: server https port (only if not 443, only with -t option)
```

Comment:
we should either read it from agent config instead, or write it in this command, but I think it shouldn't be independant

Reply:
There is no config yet, we are in the process of configuring the agent

The optional port provided within the server argument is the cfengine port, which doesn't provide the full https certificate in ssl connnections